cli/commands/quote.ts

```diff
@@ -84,13 +90,14 @@ export class Quote extends BaseCommand {
       tokenOutStr,
     ]);
 
+    // if the tokenIn str is 'ETH' or 'MATIC' or NATIVE_CURRENCY_STRING
     const tokenIn: Currency =
-      tokenInStr == 'ETH'
-        ? Ether.onChain(chainId)
+      tokenInStr in NativeCurrencyName
+        ? ExtendedEther.onChain(chainId)
         : tokenAccessor.getTokenByAddress(tokenInStr)!;
     const tokenOut: Currency =
-      tokenOutStr == 'ETH'
-        ? Ether.onChain(chainId)
+      tokenOutStr in NativeCurrencyName
+        ? ExtendedEther.onChain(chainId)
```

Comment:
i refactored interface to call `nativeOnChain` instead of `ExtendedEther.onChain` because it's confusing that method returns a class that isn't an `ExtendedEther` instance